src/java/org/apache/cassandra/config/Config.java

```diff
@@ -122,7 +122,7 @@ public static Set<String> splitCommaDelimited(String src)
     public DiskFailurePolicy disk_failure_policy = DiskFailurePolicy.ignore;
     public CommitFailurePolicy commit_failure_policy = CommitFailurePolicy.stop;
 
-    public volatile boolean use_deterministic_table_id = false;
+    public volatile boolean use_deterministic_table_id = true;
```

Comment:
Was this changed intentionally?

Reply:
temporarily yes, we'll post a patch that fixes deterministic table ids soon